RepoZ.App.Win/i18n/zh-cn.xaml

```diff
@@ -37,6 +37,7 @@
     <system:String x:Key="Open in Git Bash">在 Git Bash 中打开</system:String>
     <system:String x:Key="Open in Finder">在 Finder 中打开</system:String>
     <system:String x:Key="Open in Terminal">在 Terminal 中打开</system:String>
+    <system:String x:Key="Open {0}">打开 {0}</system:String>
```

Comment:
I don't really know about this one, @pedoc could you help out please?

Reply:
I got feedback from a non-native speaker on it being correct for mainland China and Singapore, but a check from a native speaker would be better!